htdocs/fourn/facture/card.php

```diff
@@ -456,7 +456,13 @@
 	                        $date_end=$lines[$i]->date_fin_prevue;
 	                        if ($lines[$i]->date_fin_reel) $date_end=$lines[$i]->date_fin_reel;
 	                        if ($lines[$i]->date_end) $date_end=$lines[$i]->date_end;
-
+	                        // Extrafields
+	                        if (empty($conf->global->MAIN_EXTRAFIELDS_DISABLED) && method_exists($lines[$i], 'fetch_optionals')) {
+	                        	require_once DOL_DOCUMENT_ROOT.'/core/class/extrafields.class.php';
+	                        	$extrafields = new ExtraFields($db);
```

Comment:
Because var $extrafields is already defined globally previously, it looks dangerous to use same name here.
If you rename into extrafieldsforlines, does you fix still work ?